core/src/main/java/org/infinispan/commands/read/AbstractDataCommand.java

```diff
@@ -14,22 +16,33 @@
  */
 public abstract class AbstractDataCommand extends AbstractTopologyAffectedCommand implements DataCommand {
    protected Object key;
+   private long flags;
```

Comment:
is half of a optimization... I'm pretty sure that I changed the code so ```AbstractDataCommand``` didn't extend ```AbstractFlagAffectedCommand```